flink/v1.12/flink/src/test/java/com/netease/arctic/flink/table/TestKeyed.java

```diff
@@ -441,12 +441,14 @@ public void testPartitionSinkFile() throws IOException {
   @Test
   public void testFileUpsert() throws IOException {
```

Comment:
Could you add a UT when an upsert table functionality is disabled?

Reply:
Ok